coda/templates/mdstore/bag_files_download.html

```diff
@@ -0,0 +1,6 @@
+{% extends "mdstore/base.html" %}
+{% block content %}
+    {%for item in links%}
```

Comment:
For consistency and readability, would you add a space after the first `%` and before the second one?